components/rendering/src/markdown.rs

```diff
@@ -33,12 +33,12 @@ pub struct Rendered {
 struct HeaderRef {
     start_idx: usize,
     end_idx: usize,
-    level: i32,
+    level: u32,
     id: Option<String>,
 }
 
 impl HeaderRef {
```

Comment:
can you rename all those Header structs/variable to Heading to be consistent?

Reply:
Can do!